src/9.negotiation.js

```diff
@@ -607,8 +607,13 @@ export function /*9.2.8 */SupportedLocales (availableLocales, requestedLocales,
             writable: false, configurable: false, value: subset[P]
         });
     }
-    // "Freeze" the array so no new elements can be added
-    defineProperty(subset, 'length', { writable: false });
+    
+    // Wrap in try catch for older browsers that don't support setting length of
```

Comment:
I don't see any reference in the spec about this particular operation (https://tc39.github.io/ecma402/#sec-supportedlocales), maybe we can just remove it all together. Not sure why was it added.